tests/test_normalize.rs

```diff
@@ -0,0 +1,18 @@
+use ndarray::arr1;
+use neural_zkp::normalize::*;
+
+#[test]
+fn normalize_test() {
+    let input = arr1(&[-6276474000, 8343393300, 8266027500, -7525360600, 7814137000]);
+
+    input.mapv(|x| i128::abs(x));
+
+    let Normalize::<f64> {
+        output: x,
+        n_params,
+        n_multiplications,
+        name,
+    } = normalize(input);
```

Comment:
We could test that the norm of the normalized vector is ε-close to one.

Reply:
the output of `normalize` $y \in [-1, 1]$. Should we check that it belongs to this range? it's not negligibly close to 1.